Lib/test/test_uuid.py

```diff
@@ -311,6 +311,34 @@ def test_getnode(self):
         node2 = self.uuid.getnode()
         self.assertEqual(node1, node2, '%012x != %012x' % (node1, node2))
 
+    # bpo-32502: UUID1 requires a 48-bit identifier, but hardware identifiers
+    # need not necessarily be 48 bits (e.g., EUI-64).
+    def test_uuid1_eui64(self):
+        # Reset any cached node value.
+        self.uuid._node = None
```

Comment:
Should this restore the cached value after the test is done, or at least ensure that it's set back to `None`, even if the test fails?  (E.g. add a `self.addCleanup()` or mock the global in this test.